Lib/Core/IdTCPClient.pas

```diff
@@ -433,6 +433,7 @@ procedure TIdTCPClientCustom.SetHost(const AValue: string);
   end;
   if GStack.IsIP(AValue) then begin

```

Comment:
Whoops, that code wasn't supposed to be checked in. It was just something I was playing with in another branch. I'll take it out.